source/common/json/config_schemas.cc

```diff
@@ -354,11 +354,34 @@ const std::string Json::Schema::HTTP_CONN_NETWORK_FILTER_SCHEMA(R"EOF(
 
 const std::string Json::Schema::MONGO_PROXY_NETWORK_FILTER_SCHEMA(R"EOF(
   {
-    "$schema": "http://json-schema.org/schema#",
+    "$schema" : "http://json-schema.org/schema#",
     "type" : "object",
-    "properties":{
+    "properties" : {
       "stat_prefix" : {"type" : "string"},
-      "access_log" : {"type" : "string"}
+      "access_log" : {"type" : "string"},
+      "fault" : {
+        "type" : "object",
+        "properties" : {
+          "fixed_delay" : {
+            "type" : "object",
+            "properties" : {
+              "percent" : {
+                "type" : "integer",
+                "minimum" : 0,
+                "maximum" : 100
+              },
+              "duration_ms" : {
+                "type" : "integer",
+                "minimum" : 0
```

Comment:
add `"exclusiveMinimum" : true` like the fault filter

Reply:
addressing will post in the next iteration.